top-nav-drill/index.haml

```diff
@@ -0,0 +1,43 @@
+!!! 5
+
+%html{:lang => "en"}
```

Comment:
Don't forget that you can use Ruby 1.9 style symbol keys in Haml like this: `%html{lang: "en"}`